Gruntfile.js

```diff
@@ -14,9 +14,13 @@ module.exports = function( grunt ) {
 
 	var fs = require( "fs" ),
 		gzip = require( "gzip-js" ),
+		oldNode = /^v0\./.test( process.version );
```

Comment:
I need this variable outside of this function as well (see the `load-grunt-tasks` line).